omnigibson/utils/usd_utils.py

```diff
@@ -124,7 +124,7 @@ def get_camera_params(viewport):
     view_proj_mat = helpers.get_view_proj_mat(view_params)
 
     return {
-        "pose": np.array(prim_tf),
+        "pose": np.array(prim_tf).T,
```

Comment:
hmm why transpose? maybe add a comment to explain?

Reply:
done